Client/Frontend/Browser/OpenWithSettingsViewController.swift

```diff
@@ -0,0 +1,104 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+
+class OpenWithSettingsViewController: UITableViewController {
+
+    private lazy var mailSchemes: [[String:String]] = {
+        var plistSchemes: [[String:String]] = self.loadMailSchemes()
+        return plistSchemes
+    }()
+
+    private let prefs: Prefs
+    private var currentChoice: String!
```

Comment:
Avoid using explicit unwrapped declarations like this. For example, if we tried to access currentChoice prior to viewWillAppear the app will blow up. Just use a normal optional value if the loading is deferred.